package.json

```diff
@@ -59,6 +59,8 @@
     "test:karma-chrome": "karma start --no-auto-watch --single-run --browsers=Chrome",
     "test:karma-debug": "karma start --auto-watch --no-single-run --browsers=PhantomJS_debug",
     "test:karma-dev": "karma start --auto-watch --no-single-run --capture",
-    "test:jasmine": "concurrently \"http-server -a localhost -p 8085\" \"open http://localhost:8085/test/run.html\""
+    "test:jasmine": "concurrently \"http-server -a localhost -p 8085\" \"open http://localhost:8085/test/run.html\"",
+    "preversion:bower": "sed 's/\"version\": \"[^,]*\"/\"version\": \"'$npm_package_version'\"/' bower.json  > output &&  mv output bower.json",
+    "version": "npm run preversion:bower && git add -A bower.json"
```

Comment:
@kriskowal  added preversion script to keep bower.json & package.json in sync

Usage: 
```
$ npm version patch
v5.0.8

> collections@5.0.8 version ~/collections
> npm run preversion:bower && git add -A bower.json


> collections@5.0.8 preversion:bower ~/collections
> sed 's/"version": "[^,]*"/"version": "'$npm_package_version'"/' bower.json  > output &&  mv output bower.json
```

Result: 
```
$ git show
commit b42e78caaa1c341eb70b44bd0bec1f7b862d047e (HEAD -> bower_support, tag: v5.0.8)
Author: cesine <cesine@yahoo.com>
Date:   Wed Nov 22 21:47:51 2017 -0500

    5.0.8

diff --git a/bower.json b/bower.json
index b8e43ce..eb4bdaa 100644
--- a/bower.json
+++ b/bower.json
@@ -1,7 +1,7 @@
 {
   "name": "collections",
   "main": "collections.min.js",
-  "version": "5.0.7",
+  "version": "5.0.8",
   "homepage": "http://www.collectionsjs.com/",
   "author": "Kris Kowal <kris@cixar.com> (http://github.com/kriskowal)",
   "description": "data structures with idiomatic JavaScript collection interfaces",
diff --git a/package.json b/package.json
index db66efe..a87b0f7 100644
--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "collections",
-  "version": "5.0.7",
+  "version": "5.0.8",
   "description": "data structures with idiomatic JavaScript collection interfaces",
   "homepage": "http://www.collectionsjs.com",
   "author": "Kris Kowal <kris@cixar.com> (http://github.com/kriskowal)",
```